Site/IdentifierConfig.php

```diff
@@ -5,7 +5,13 @@
 
 final class IdentifierConfig
 {
+    /**
+     * @var array
+     */
     private $hostnames;
+    /**
+     * @var array
```

Comment:
minor: Voegt deze PHPDoc nog iets toe? Ik voeg zelf dit soort PHPDoc niet meer toe aangezien de constructor een een typehint heeft en PHPStorm hierdoor toch het juiste type pakt.

Reply:
als we deze toevoegen, kunnen we later met rector php hier vrij snel typed properties van maken: https://tomasvotruba.com/blog/2018/11/15/how-to-get-php-74-typed-properties-to-your-code-in-few-seconds/

Reply:
Nice!